impl/c-qube/src/services/csv-adapter/parser/utils/csvreader.ts

```diff
@@ -26,7 +33,17 @@ export async function readCSVFile(filePath: string): Promise<string[]> {
   const fileContent = await fs.readFile(filePath, 'utf-8');
 
   return fileContent
-    .split('\n')
-    .map((row: string) => row.trim())
-    .filter((row: string) => row !== '');
+  .split('\n')
+  .map((row: string) => row.trim())
+  .filter((row: string) => row !== '');
 }
+
+export function getCsvDelimiter(configPath: string): string | undefined {
+  try {
+    const configContent = fs1.readFileSync(configPath, 'utf-8');
+    const config = JSON.parse(configContent);
+    return config.globals.csvDelimiter;
```

Comment:
handle errors here as well, there is no gurantee that this particular element is defined or not. Also make sure you are handling the case of this function returning as `undefined` wherever it is being being and defaulting to a `','` in such cases.